gobblin-core/src/main/java/gobblin/source/extractor/extract/sftp/SftpFsHelper.java

```diff
@@ -43,7 +47,7 @@
  */
 public class SftpFsHelper implements SizeAwareFileBasedHelper {
```

Comment:
Can you add some explanation on  how sftp helper works? I see channels, and channelexec, but without knowing more about Sftp it's hard to understand how all of this fits together.